src/main/java/mate/academy/mapper/BookMapper.java

```diff
@@ -11,4 +12,26 @@ public interface BookMapper {
     BookDto toDto(Book book);
 
     Book toModel(CreateBookRequestDto requestDto);
+
+    default Book toModel(Book book, BookUpdateDto requestDto) {
+        if (requestDto.title() != null) {
+            book.setTitle(requestDto.title());
+        }
+        if (requestDto.author() != null) {
+            book.setAuthor(requestDto.author());
+        }
+        if (requestDto.isbn() != null) {
+            book.setIsbn(requestDto.isbn());
+        }
+        if (requestDto.coverImage() != null) {
+            book.setCoverImage(requestDto.coverImage());
+        }
+        if (requestDto.price() != null) {
+            book.setPrice(requestDto.price());
+        }
+        if (requestDto.description() != null) {
+            book.setDescription(requestDto.description());
+        }
+        return book;
+    }
```

Comment:
Not fully sure but something like that may work, please recheck
```suggestion
    Book toModel(@MappingTarget Book book, BookUpdateDto requestDto);
```

Reply:
I checked and this way it doesn't work (console writes that params can't be null when mapping to dto from model that was mapped the way you propose (as I understand, null fields get set to book too))